mastodon/src/main/res/values-de-rDE/strings_sk.xml

```diff
@@ -136,4 +136,7 @@
     <string name="sk_clear_recent_emoji">Kürzlich verwendete Emojis löschen</string>
     <string name="sk_disable_relocate_publish_button_to_enable_customization">Deaktiviere die Funktion "Veröffentlichen Schaltfläche verschieben", um eine Anpassung zu ermöglichen.</string>
     <string name="sk_keep_only_latest_notification">Nur neueste Benachrichtigung behalten</string>
-</resources>
+    <string name="sk_poll_option_add">Neue Umfrageoption hinzufügen</string>
+    <string name="sk_fab_compose">Fehler beim Veröffentlichen</string>
```

Comment:
This is duplicated, with the line below. I think something like this would fit better.
```suggestion
    <string name="sk_fab_compose">Verfassen</string>
```

Reply:
Thanks for the tip. I made a mistake there. I have just adjusted it.